packages/create-emotion/src/sheet.js

```diff
@@ -35,49 +36,61 @@ function sheetForTag(tag) {
   }
 }
 
-function makeStyleTag() {
+function makeStyleTag(nonce?: string): HTMLStyleElement {
   let tag = document.createElement('style')
   tag.type = 'text/css'
   tag.setAttribute('data-emotion', '')
+  if (nonce !== undefined) {
+    tag.setAttribute('nonce', nonce)
+  }
   tag.appendChild(document.createTextNode(''))
+  // $FlowFixMe
   document.head.appendChild(tag)
   return tag
 }
 
 export default class StyleSheet {
-  constructor() {
+  injected: boolean
+  isSpeedy: boolean
+  isBrowser: boolean
+  ctr: number
+  sheet: string[]
+  tags: HTMLStyleElement[]
+  nonce: string | void
+  constructor(nonce?: string) {
```

Comment:
Should we make this an object in case we need more options later.

Reply:
I don't think we need to since it's an internal API.